frontend/src/component/common/NewConstraintAccordion/ConstraintAccordionEdit/ConstraintAccordionEditBody/FreeTextInput/FreeTextInput.tsx

```diff
@@ -87,7 +87,7 @@ export const FreeTextInput = ({
     };
 
     return (
-        <div>
+        <>
```

Comment:
`<>` is pretty much a shortcut to `<Fragment>`, a logic-only (no markup) element that serves as a grouper/aggregator of elements: https://react.dev/reference/react/Fragment

The reason why I opted to switch this `div` for a fragment was the fact that other sibling Inputs used the fragment instead of the div, and so the styling I was applying through my wrappers was not being consistent.

The simple `div` was not doing anything anyways, so this makes no difference in other usages of this component.